<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>473</width>
    <height>530</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>التهيئة</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="lblTermId">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lblRowNumber">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="4" column="0">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="7" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_6"/>
       </item>
       <item row="3" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="styleSheet">
            <string notr="true">	font: 10pt &quot;PT Bold Heading&quot;;</string>
           </property>
           <property name="text">
            <string>الماده :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboSubjects">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>35</height>
            </size>
           </property>
           <property name="layoutDirection">
            <enum>Qt::RightToLeft</enum>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color:rgb(255, 255, 255);
color:rgb(0, 0, 0);
font: 10pt &quot;Motken K Sina&quot;;
    border-radius:10px;
	
	border: 1px solid #5b586e;

</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="pushButton_3">
           <property name="styleSheet">
            <string notr="true">
background-color:qlineargradient(spread:pad, x1:0.506975, y1:0.432, x2:0.99, y2:0.437227, stop:0.900498 rgba(4, 0, 62, 255), stop:1 rgba(255, 255, 255, 255));
color:rgb(255, 255, 255);
 font: 75 15pt &quot;Motken K Sina&quot;;
</string>
           </property>
           <property name="text">
            <string>تعديل بيانات الحصص </string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="9" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QPushButton" name="btnCancelUpdateSession">
           <property name="maximumSize">
            <size>
             <width>140</width>
             <height>60</height>
            </size>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">


QPushButton {
background-color:rgb(50, 65, 75);
 
 color:rgb(255, 255, 255);
font: 75 12pt &quot;Motken K Sina&quot;;

  border-radius:10px;
	padding: 10px 5;
	
}

QPushButton:hover {
background-color:rgb(191, 194, 181);
      	 color:rgb(0, 0, 0);
		font: 75 10pt &quot;Motken K Sina&quot;;

       

}</string>
           </property>
           <property name="text">
            <string>إلغاء</string>
           </property>
           <property name="icon">
            <iconset resource="icons_rc/icons.qrc">
             <normaloff>:/icons_qrc/exclamation-48.ico</normaloff>:/icons_qrc/exclamation-48.ico</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnSaveTermSessionsUpdate">
           <property name="maximumSize">
            <size>
             <width>140</width>
             <height>60</height>
            </size>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">


QPushButton {
background-color:rgb(50, 65, 75);
 
 color:rgb(255, 255, 255);
font: 75 12pt &quot;Motken K Sina&quot;;

  border-radius:10px;
	padding: 10px 5;
	
}

QPushButton:hover {
background-color:rgb(191, 194, 181);
      	 color:rgb(0, 0, 0);
		font: 75 10pt &quot;Motken K Sina&quot;;

       

}</string>
           </property>
           <property name="text">
            <string>حفظ</string>
           </property>
           <property name="icon">
            <iconset resource="icons_rc/icons.qrc">
             <normaloff>:/icons_qrc/true.png</normaloff>:/icons_qrc/true.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>35</width>
             <height>35</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="styleSheet">
            <string notr="true">	font: 10pt &quot;PT Bold Heading&quot;;</string>
           </property>
           <property name="text">
            <string>الصف :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboClasses">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>35</height>
            </size>
           </property>
           <property name="layoutDirection">
            <enum>Qt::RightToLeft</enum>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color:rgb(255, 255, 255);
color:rgb(0, 0, 0);
font: 10pt &quot;Motken K Sina&quot;;
    border-radius:10px;
	
	border: 1px solid #5b586e;

</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="5" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QPlainTextEdit" name="txtNumberOfSessions">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>35</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPlainTextEdit {
    background-color:rgb(255, 255, 255);
    font: 75 8pt &quot;Motken K Sina&quot;;
    color:rgb(0, 0, 0);
    border-radius:10px;
	padding: 2px 0;
	border: 1px solid #5b586e;

}</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>140</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">	font: 10pt &quot;PT Bold Heading&quot;;</string>
           </property>
           <property name="text">
            <string>عدد الحصص  :</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="8" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>txtNumberOfSessions</tabstop>
  <tabstop>pushButton_3</tabstop>
  <tabstop>btnCancelUpdateSession</tabstop>
  <tabstop>btnSaveTermSessionsUpdate</tabstop>
  <tabstop>comboClasses</tabstop>
  <tabstop>comboSubjects</tabstop>
 </tabstops>
 <resources>
  <include location="icons_rc/icons.qrc"/>
 </resources>
 <connections/>
</ui>
